src/main/resources/com/cloudbees/hudson/plugins/folder/AbstractFolder/tasks-new_ja.properties

```
@@ -20,4 +20,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-delete={0}\u306e\u524a\u9664
+delete={0}\u306E\u524A\u9664
```

```suggestion
delete={0}\u306e\u524a\u9664
```

just to leave it untouched